schemas/paths/events.yaml

```diff
@@ -14,15 +14,11 @@ get:
 
     If you don't provide `start` or `end` parameters, the default search range is the **last 7 days**.
 
-    ### Filtering events with the`suspect` flag
+    ### Filtering events with the `suspect` flag
 
     The `/v4/events` endpoint unlocks a powerful method for fraud protection analytics. The `suspect` flag is exposed in all events where it was previously set by the update API.
 
     You can also apply the `suspect` query parameter as a filter to find all potentially fraudulent activity that you previously marked as `suspect`. This helps identify patterns of fraudulent behavior.
-    
-    ### Environment scoping
```

Comment:
This is unexpected, why is this improvement not in koala?

Reply:
It was introduced only in the schema for readme.com that was manually prepared initially. I decided that it is important and added it to this schema.

We should add it to the Koala to keep synced.